osc/commandline.py

```diff
@@ -7012,6 +7012,8 @@ def parse_repoarchdescr(self, args, noinit=False, alternative_project=None, igno
                   help='define macro X with value Y')
     @cmdln.option('--build-opt', metavar='OPT', action='append',
                   help='pass option OPT to the build command')
+    @cmdln.option('--buildtool-opt', metavar='OPT', action='append',
+                  help='pass option OPT to the build command')
```

Comment:
This is currently printed as
```
    --buildtool-opt=OPT
                        pass option OPT to the build command
    --build-opt=OPT     pass option OPT to the build command
```

The help text clashes with  the help from the `--build-opt=OPT`. It should be better `help='pass option OPT to the build tool command'`

Reply:
Good catch, thanks!